app/controllers/apply_job_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ApplyJobController < ApplicationController
+
+  def new
+    url = request.url 
+    job_id = url.scan(/[0-9]{1,8}$/).last.to_i    
```

Comment:
Truy cập params của request thông qua `ActionController::Parameters` bằng hàm `params`

```suggestion
    job_id = params[:job_id]
```